src/main/java/PageFactory.java

```diff
@@ -0,0 +1,12 @@
+import Pages.*;
+import java.io.IOException;
+import static com.codeborne.selenide.Selenide.open;
+
+public class PageFactory {
+
+    public static LoginPage openLoginPage() {
+
```

Comment:
try to avoid such unneeded empty spaces, it will quickly become messy once you have more code